### PR TITLE
fix: dyups module doc updated

### PR DESCRIPTION
**Pull request by zaozaoniao**
issue： [ngx_http_dyups_module module doc is obsoleted](https://github.com/alibaba/tengine/issues/1671)

### CODE REVIEW

File: docs/modules/ngx_http_upstream_dyups_module.md [hunk 1]
```diff
@@ -77,19 +77,19 @@ file: conf/upstream.conf
...
 
 * Install dyups module with lua-nginx-module and upstream check module
     * upstream check module: To make upstream check module work well with dyups module, you should use `./modules/ngx_http_upstream_check_module`.
-    * lua-nginx-module: To enable [dyups LUA API](#lua-api-example), you MUST put `--add-module=./modules/ngx_http_lua_module` in front of `--add-module=./modules/ngx_http_dyups_module` in the `./configure` command.
+    * lua-nginx-module: To enable [dyups LUA API](#lua-api-example), you MUST put `--add-module=./modules/ngx_http_lua_module` in front of `--add-module=./modules/ngx_http_upstream_dyups_module` in the `./configure` command.
 
 ```bash
 # to compile as a static module
-$ ./configure --add-module=./modules/nginx_upstream_check_module --add-module=./modules/ngx_http_lua_module --add-module=./modules/ngx_http_dyups_module
+$ ./configure --add-module=./modules/nginx_http_upstream_check_module --add-module=./modules/ngx_http_lua_module --add-module=./modules/ngx_http_upstream_dyups_module
```
Comment (chobits) [inline docs/modules/ngx_http_upstream_dyups_module.md:92]: ./modules/nginx_http_upstream_check_module -> modules/ngx_http_upstream_check_module/

Comment (zaozaoniao) [reply docs/modules/ngx_http_upstream_dyups_module.md]: ok

**Comment by chobits** [comment]
If no other changes, you need to [squash all commits  into one](https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together). Then merged commit log will be clean.

(may git rebase -i .. && git push --force )

**Comment by zaozaoniao** [comment]
@chobits  done

**Pull request merged**